src/darwin/Framework/CHIP/templates/MTRAttributeUtils-src.zapt

```diff
@@ -0,0 +1,47 @@
+{{> header excludeZapComment=true}}
+
+#import "MTRAttributeUtils.h"
+
+#include <app-common/zap-generated/ids/Attributes.h>
+#include <app-common/zap-generated/ids/Clusters.h>
+
+using namespace chip;
+using namespace chip::app;
+
+{{#zcl_clusters}}
+{{#if (isSupported (asUpperCamelCase name preserveAcronyms=true))}}
+static BOOL MTRAttributeIsSpecifiedIn{{asUpperCamelCase name preserveAcronyms=true}}Cluster(AttributeId aAttributeId)
+{
+    using namespace Clusters::{{asUpperCamelCase name}};
+    switch (aAttributeId) {
+        {{#zcl_attributes_server removeKeys='isOptional'}}
```

Comment:
```suggestion
        {{#zcl_attributes_server}}
```
You don't care about the optionality, since you are not doing any value processing here.